apps/music/js/utils.js

```diff
@@ -75,13 +77,13 @@ function createListElement(option, data, index, highlight) {
       a.dataset.keyRange = 'all';
       a.dataset.option = data.option;
 
-      li.appendChild(titleSpan);
+      a.appendChild(titleSpan);
```

Comment:
Does the list elements also use `<a>` in the other apps? I remember why we use `<li>` was because the building blocks suggested the apps to use `<li>` for list elements.